shared/templates/static/oval/sshd_expected.xml

```diff
@@ -0,0 +1,72 @@
+<def-group>
```

Comment:
This oval file does not follow the existing convention of one definition class per OVAL file. These can and should be broken out into separate files. In addition, some of the logic can be moved from the SSH OVAL files into the broken ssh expected files which would make the SSH OVAL files cleaner to read/easier to understand and simpler.